python_modules/libraries/dagster-fivetran/dagster_fivetran/assets.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Optional
+
+from dagster import AssetKey, Out, Output
+from dagster.core.asset_defs import AssetsDefinition, multi_asset
+from dagster.utils.backcompat import experimental
+from dagster_fivetran.resources import DEFAULT_POLL_INTERVAL
+from dagster_fivetran.utils import generate_materializations
+
+
+@experimental
+def fivetran_assets_factory(
+    connector_id: str,
+    asset_keys: List[AssetKey],
```

Comment:
Something to think about here is that we might ultimately want to offer the ability to include metadata about these assets.  E.g. a user might know the set of columns they contain and want to include that on the asset definitions.

Reply:
yeah I wonder if that could be addressed with something similar to the runtime_metadata_fn that we provide w/ the dbt SDA integration. It's not perfect, because you might know something before runtime and want to represent that without having to run the thing, but it is the most flexible solution.

Reply:
The runtime aspect didn't even occur to me, but yes, that too.